[FIX][web_m2x_options] Fix Qweb templates.

yajo opened the pull request:
They were trying to replace non-existing elements, and this was being logged to console:

    Can't find "a.oe_m2o_cm_button" when extending template FieldMany2One
    Can't find "span.badge" when extending template FieldMany2ManyTag

@Tecnativa

yajo committed:
[FIX][web_m2x_options] Fix Qweb templates.

They were trying to replace non-existing elements, and this was being logged to console:

    Can't find "a.oe_m2o_cm_button" when extending template FieldMany2One
    Can't find "span.badge" when extending template FieldMany2ManyTag

pedrobaeza commented on web_m2x_options/__openerp__.py:
```diff
@@ -2,7 +2,7 @@
 {
     "name": 'web_m2x_options',
-    "version": "9.0.1.0.0",
+    "version": "9.0.1.0.1",
```
Don't you need to update?

yajo replied:
Update what? The addon?

pedrobaeza replied:
Yes, I mean that if it's the case, you need to put 9.0.1.1.0

yajo committed:
Raise version correctly.

yajo commented:
It turns out that making the addon "work" makes `many2many_tags` widget fail. Gonna fix.

yajo committed:
[FIX] Make many2many_tags tag deletion work again

yajo committed:
Correct replacement of event

Instead of overwriting all events from upstream widget, better just overwrite the one you need and inherit the rest.

yajo commented:
This should be OK now. Tested it works in production.

njeudy reviewed (approved):
- test and code review

rafaelbn reviewed (approved):
thanks
